types/logger.d.ts

```diff
@@ -19,13 +19,12 @@ export type Bindings = pino.Bindings
 
 export type ChildLoggerOptions = pino.ChildLoggerOptions
 
-export type FastifyLoggerInstance = pino.Logger
-// TODO make pino export BaseLogger again
-// export type FastifyBaseLogger = pino.BaseLogger & {
-export type FastifyBaseLogger = pino.Logger & {
+export type FastifyBaseLogger = pino.BaseLogger & {
   child(bindings: Bindings, options?: ChildLoggerOptions): FastifyBaseLogger
 }
 
+export type FastifyLoggerInstance = FastifyBaseLogger
```

Comment:
I suggest to delete this type. What do you think @mcollina ?

Reply:
Add a TODO comment to delete it in the next major

Reply:
Using jsdoc @deprecated will make intelliSense show it stroked indicating its deprecation.tsd has imho isDeprecated Check which can be utilited

Reply:
I agree, no need to break backwards compatibility right now. I've added the todo comment.